### ISSUE TITLE
Collect the white paper workflow from creation to publication

### ISSUE THREAD

**Issue by catlook** [rackerlabs/rackspace-how-to#1671]
...  from Dominic Smith,and discuss how it might change when we come on board for technical papers

**Comment by catlook** [comment]
Requested meeting:

Dominic,

Tl;dr – I’d like a half hour meeting with you (vidyo) this week, if possible, to discuss your marketing whitepaper submission/publication process.  I’m working on the technical whitepaper side and want collaborate and consolidate.  Shall I find an open spot on your calendar?  Do you have a preference?  Is there someone else I should work with or include?

The whole story:

I understand you’re the guy who handles marketing whitepapers.  Could you send me any existing templates or guides you have for would-be contributors to your whitepaper collection?  Or point me to the best source of info?  I’d also like to discuss the submission and publication process.  What’s the workflow?

I’ve been asked to create a similar process for technical whitepapers (which may have both internal and external contributors, potentially).  Laura Clymer told me that it’s best that there is a single point of contact for all whitepapers, which would likely be you or your team.  If a technical paper comes in, you guys would then kick it over to me or my team for content review, validation, reworking, and so on.  But what happens then?  Do we send back to you for publication? Or have our own process?  Will the papers live in the same place or not?

We don’t need to have all the answers, but I’d love to sit with you for 30 minutes or less this week to talk about these things and see what else comes up.  The idea is to use the existing process and not reinvent the wheel where possible.  And split things into a similar but distinct (or wildly different) workflow where appropriate.

Let me know if you ave any questions.

**Comment by catlook** [comment]
Meeting set for 1pm on Friday, Dec 2.

**Comment by catlook** [comment]
Notes from meeting:
 
D's team uses Kapost, content creation and repo, for their whitepapers. It is available at will, internally.  Individual articles may be shared with external users.
 
To see the papers, use following link:
https://rackspace.kapost.com/gallery/collection-groups/ed72e87c-2b70-4d4f-8de4-1410ec055ec7
- Choose "Rackspace Main Public"
- Search for "whitepapers"
 
To request new content, use following link:
https://rackspace.kapost.com/ideas/new
 
On D's team, following folks are decision makers for requesting time or resources:
shanon.montelongo
kelly.hopping
 
About Kapost:
- There are 150 contributor licenses. 
- Content approvers do NOT need a license, if they are just yes/no.  They can be emailed a link to the content under review.  If they went to suggest changes, they would have to email their comments. 
- Non-licensed folks can't change content in Kapost.
- Kapost is NOT intended to be a customer-facing portal. Article links can be sent to customers or prospects.
 
Process today:
- Most content is written by writers rather than submitted from others.  More content maybe outsourced to third-party writers or purchased from other sources.  Kapost provides the means to SME approve and stake-holder approve content.  [Don't know if they pass it by an editor also.]
- Once the paper content is good, it is passed to layout and design team.  They have templates and tools to produce a Rackspace-branded whitepaper with a consistent look and feel, and the submitted content.
- They deliver PDFs.
 
Future:
- D agrees that his team may be willing to serve as a central point of contact for whitepaper requests and funnel the tech papers to us.  Depends on the workload.
- Ideally, the M&S papers would be readily available to the public, rather than on request.  Certainly, tech papers need to be public-facing.
- It's possible that the .com team, which is currently considering redesign options, could add a whitepaper hub for PDF content.  If they do, this could house tech and M&S whitepapers.
 
ToDo:
D- talk to Shannon and Kelly about his team being point of contact for all papers
C- talk to Brian Law on .com about the whitepaper hub idea.
